app/pages/select/controller.js

```diff
@@ -0,0 +1,35 @@
+import { getRelatedOrganisations } from '../../helpers/api/oapi/getOrganisation';
+
+export const getProxyOrganisations = async ({ accessToken, orgId }) => {
+  const organisationsList = await getRelatedOrganisations({ accessToken, orgId });
+  return organisationsList;
+};
+
+export const getSelectContext = async ({ accessToken, orgId, orgName }) => {
+  const organisationsList = await getProxyOrganisations({ accessToken, orgId });
+
+  let radioList;
+
+  if (organisationsList) {
+    radioList = organisationsList.map((org) => ({ value: org.organisationId, text: org.name }));
+    radioList.sort((a, b) => a.text.localeCompare(b.text));
+  }
```

Comment:
I think this might sit nicely in a separate function, e.g. `formatOrganisationList` or `transformOrganisationList` or something similar.